src/Strategy/AbstractStrategy.php

```diff
@@ -59,7 +59,7 @@ protected function invokeController($controller, array $vars = [])
      * Attempt to build a response
      *
      * @param  mixed $response
-     * @return \League\Http\ResponseInterface
+     * @return Response
```

Comment:
Can this be FQN please. I'm aware it doesn't need to be but the rest of the package is.